src/python/dxpy/scripts/dx.py

```diff
@@ -2652,8 +2652,8 @@ def handle_arg_conflicts(args):
         if args.mode in ("globalworkflow", "applet", "app") and args.keep_open:
             build_parser.error("Global workflows, applets and apps cannot be kept open")
 
-        if args.nextflow and args.repository and not args.remote:
-            build_parser.error("Cannot build Nextflow pipeline from repository locally. Please use --remote.")
+        if args.nextflow and args.repository and args.remote:
+            build_parser.error("Nextflow pipeline built from repository is always build using Nextflow Pipeline Importer. This is not compatible with --remote.")
```

Comment:
Would this suggestion be more unambiguous?

```suggestion
            build_parser.error("Nextflow pipeline built from a remote Git repository is always build using the Nextflow Pipeline Importer app. This is not compatible with --remote.")
```

Reply:
Fixed

Reply:
Fixed